src/link_manager/LinkManagerStateMachineActiveActive.cpp

```diff
@@ -176,6 +176,8 @@ void ActiveActiveStateMachine::handleMuxStateNotification(mux_state::MuxState::L
     MUXLOGWARNING(boost::format("%s: state db mux state: %s") % mMuxPortConfig.getPortName() % mMuxStateName[label]);
 
     mWaitTimer.cancel();
+
+    mLastMuxNotificationType = LastMuxNotificationType::MuxNotificationFromToggle;
```

Comment:
If the mux notification is not posted to mux state machine (`postMuxStateEvent(label)`), should we still update `mLastMuxNotificationType`?

Reply:
This is to align with the update of `mLastMuxStateNotification`